tilelang/intrinsics/mma_sp_macro_generator.py

```
@@ -294,15 +294,16 @@ def _warp_ldmatrix_a(
                 )
 
                 if ldmatrix_available:
+                    row_off, col_off = get_ldmatrix_offset("A", tx, 0, stride, a_dtype, a_transposed)
+                    if a_transposed:
+                        src_indices = tuple(A_other) + (A_base0 + wk + row_off, A_base1 + wi + col_off)
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)